llvm/lib/Target/RISCV/RISCVFeatures.td

```diff
@@ -52,108 +61,118 @@ class RISCVExperimentalExtension<string name, int major, int minor, string desc,
 
 def FeatureStdExtI
     : RISCVExtension<"i", 2, 1,
-                     "'I' (Base Integer Instruction Set)">;
+                     "'I' (Base Integer Instruction Set)">,
+                     RISCVExtensionBitmask<0, 0>;
 
 def FeatureStdExtE
     : RISCVExtension<"e", 2, 0,
-                     "Implements RV{32,64}E (provides 16 rather than 32 GPRs)">;
+                     "Implements RV{32,64}E (provides 16 rather than 32 GPRs)">,
+                     RISCVExtensionBitmask<0, 1>;
 
 def FeatureStdExtZic64b
     : RISCVExtension<"zic64b", 1, 0,
-                     "'Zic64b' (Cache Block Size Is 64 Bytes)">;
+                     "'Zic64b' (Cache Block Size Is 64 Bytes)">,
+                     RISCVExtensionBitmask<0, 2>;
```

Comment:
This isn't a very interesting feature to multiversion on. Will hwprobe ever support this?